<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeatSelectionForm</class>
 <widget class="QDialog" name="SeatSelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>931</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Seat Selection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/Images/new box office transaction pic.png</normaloff>:/Images/new box office transaction pic.png</iconset>
  </property>
  <widget class="QWidget" name="widTheater" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1350</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblTheater">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="MyResources.qrc">:/Images/movie theater.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ptbSelectSeats">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>890</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select seats</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptbCancel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>890</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
